### PR TITLE
Card component

### PR DESCRIPTION
**Pull request by tanishk15sharma**
**Card Component branch**

Animation

-  Basic card
-  Basic W/ Header 
-  Basic w/ Text over Media

Animation2

-  Horizontal Image and Text

### CODE REVIEW

**Comment by ashishnnnnn** [comment]
Code looks fine :+1:

**Comment by iamalhera** [comment]
looks right

File: docs.html [hunk 1]
```diff
@@ -373,7 +373,165 @@ <h2 class="list-topic">Components</h2>
...
+
+<div class="component-subhead" >Basic Card</div>
+
+
+<div class="card-container">
+  <img
+    src="https://picsum.photos/380/240"
+    alt="nature img"
+    class="random-img"
+    loading="eager"
```
Comment (junaidshaikh-js) [inline docs.html:392]: the `loading="lazy"`  attribute should be used for the images which are not in the viewport at initial loading, it improves the performance of the website.

File: docs.html [hunk 2]
```diff
@@ -373,7 +373,165 @@ <h2 class="list-topic">Components</h2>
...
+    <button class="card-btn">READ</button>
+    <button class="card-btn">BOOKMARK</button>
+    <i class="far fa-heart card-icon"></i>
+    <i class="fas fa-share-alt card-icon"></i>
+    <i class="fas fa-ellipsis-v card-icon"></i>
+  </div>
+</div>
+
+
+
```
Comment (junaidshaikh-js) [inline docs.html:412]: remove unnecessary white lines from code, using some code formatter will solve the problem

File: style/components/card.css [hunk 3]
```diff
@@ -0,0 +1,98 @@
+img {
```
Comment (junaidshaikh-js) [inline style/components/card.css:1]: can be moved to reset.css if you have one

File: style/components/joinComponent.css [hunk 4]
```diff
@@ -2,3 +2,4 @@
 @import url("./alert.css");
 @import url("./button.css");
 @import url("./badge.css");
+@import url("./card.css");
```
Comment (junaidshaikh-js) [inline style/components/joinComponent.css:5]: joinComponent.css can be named something else like style.css

**Review by junaidshaikh-js — COMMENTED** [review]
Format your code using code formatter.

File: style/components/card.css (hunk 3 above)
Comment (tanishk15sharma) [reply style/components/card.css:1]: sure

File: docs.html (hunk 1 above)
Comment (tanishk15sharma) [reply docs.html:392]: thank you ,  doing the changes

File: docs.html (hunk 2 above)
Comment (tanishk15sharma) [reply docs.html:412]: okay I will find a code formatter to solve this

**Pull request merged**